generator/src/main/java/com/scottlogic/deg/generator/utils/FileUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.scottlogic.deg.generator.utils;
+
+import java.text.DecimalFormat;
+import java.util.Arrays;
+
+public class FileUtils {
```

Comment:
Make this `static`?
@mrspaceman are you aware of this change, it might have an impact on your changes.

Reply:
Should this be a dependency so it allows for variance in future.

Reply:
You can't mark classes as static in Java and I couldn't think of when you'd want alternate implementations of the formatter hence not a dependency.

Reply:
Does it support testing if you can mock this type and ignore the implementation. If so then it would be better if it's a dependency

Reply:
it's more testable than it was as a private class, it's not a downgrade to make it publicly available if it's needed elsewhere.

this isn't the sort of thing that needs to be mocked.

the only real future problem is if some other method of modifying the filenames is required for some reason, but it can be changed at that point.

Reply:
I'll leave it as static as right now we don't need different implementations of this. In the future if we do need it this can be a dependency.